### PR TITLE
Fix the placeholders in the formbuilder. Fixes #1559

### LINKED ISSUE #1559: Placeholders formbuilder
There is a bug for having placeholders in textfields etc. 
@WouterSioen said that the rendering of the form tags has changed. It cloud be some issue in there.

### PR DESCRIPTION
**Pull request by WouterSioen**
They weren't used.

### CODE REVIEW

**Comment by carakas** [comment]
👍

File: src/Frontend/Modules/FormBuilder/Widgets/Form.php [hunk 1]
```diff
@@ -214,6 +214,9 @@ private function loadForm()
                     if (isset($field['validations']['email'])) {
                         $txt->setAttribute('type', 'email');
                     }
+                    if ($item['placeholder']) {
```
Comment (carakas) [inline src/Frontend/Modules/FormBuilder/Widgets/Form.php]: is it always available or would an isset like above be better ?

File: src/Frontend/Modules/FormBuilder/Widgets/Form.php [hunk 2]
```diff
@@ -271,6 +274,9 @@ private function loadForm()
                     if ($item['required']) {
                         $txt->setAttribute('required', null);
                     }
+                    if ($item['placeholder']) {
```
Comment (carakas) [inline src/Frontend/Modules/FormBuilder/Widgets/Form.php]: idem

File: src/Frontend/Modules/FormBuilder/Widgets/Form.php (hunk 1 above)
Comment (WouterSioen) [inline src/Frontend/Modules/FormBuilder/Widgets/Form.php]: it's always set (on line 151)

File: src/Frontend/Modules/FormBuilder/Widgets/Form.php (hunk 2 above)
Comment (WouterSioen) [inline src/Frontend/Modules/FormBuilder/Widgets/Form.php]: it's always set (on line 151)

Comment (carakas) [inline src/Frontend/Modules/FormBuilder/Widgets/Form.php]: in that case :shipit:

**Pull request merged**